src/main/scala/com/fulcrumgenomics/sv/AlignedSegment.scala

```diff
@@ -19,13 +19,16 @@ import scala.collection.{BitSet, mutable}
  * @param positiveStrand true if the read was mapped to the positive strand, false otherwise
  * @param cigar          the cigar **in sequencing order**
  * @param range          the genomic range to which this alignment maps
+ * @param recs           the records from which this aligned segment originated from.  May contain multiple if two
+ *                       aligned segments were merged, and be empty if not tracking.
  */
 case class AlignedSegment(origin: SegmentOrigin,
                           private val readStart: Int,
                           private val readEnd: Int,
                           positiveStrand: Boolean,
                           private val cigar: Cigar,
-                          range: GenomicRange) {
+                          range: GenomicRange,
+                          recs: Seq[SamRecord] = IndexedSeq.empty) {
```

Comment:
Could make this `Iterable[SamRecord]` and then `Some(rec)` and `None` are valid values.

Reply:
leaving it as is as it breaks tests even though we have the same contents in the iterable and I don't want to spend the time figuring it out.